configs/example/se.py

```diff
@@ -278,5 +278,12 @@ def get_processes(options):
     MemConfig.config_mem(options, system)
     config_filesystem(system, options)
 
+for cpu in system.cpu:
+    cpu.createInterruptController()
+    #cpu.connectAllPorts(system.membus)
+    cpu.interrupts[0].pio = system.membus.master
```

Comment:
I think you need to do this for each thread. E.g.,

```python
for cpu in system.cpu:
    cpu.createInterruptController()
    for i in range(cpu.numThreads):
        cpu.interrupts[i].pio = system.membus.master
        ....
```

Reply:
I still get the same error.